main.cpp

```diff
@@ -107,6 +128,76 @@ Thing 4) analog console
     3) add tone
  */
 
+/*
+Thing 5) Mixer
+5 properties:
+    1) faders 
+    2) routing panel 
+    3) record ready btn
+    4) input fx
+    5) input selector
```

Comment:
These are objects, not primitives!   You need to pick things that can be represented with `int float double bool char std::string`
- [ ] revise